pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -382,10 +382,12 @@
   call_to_action_get_started: "Get started"
   call_to_action_view_our_supporters: "View our supporters"
   call_to_action_explore_self_paced_pl: "Explore self-paced professional learning"
+  call_to_action_explore_k5_self_paced_pl: "Explore K-5 self-paced professional learning"
```

Comment:
Nit: can you move these two strings [to this section](https://github.com/code-dot-org/code-dot-org/blob/ada66673bbcf8c1330803c1889a6e8b534d22d9c/pegasus/sites.v3/hourofcode.com/i18n/en.yml#L401) to follow the naming pattern?

Reply:
ahh yes good call! Updated!